src/include/fips.h

```diff
@@ -20,6 +20,14 @@
 extern int fips;			/* module status */
 
 
+/*
+ * Perform an integrity check on libica.so by calculating an HMAC from
+ * the file contents using a static HMAC key, and comparing it to a
+ * pre-calculated HMAC in a separate file. The HMAC key and HMAC file
+ * may be provided by a Distributor when building the packet.
+ */
+void fips_lib_integrity_check(void);
```

Comment:
fips_lib_integrity_check is only used in fips.c, so you can remove the prototype from fips.h and make it static in fips.c.

Reply:
Done.